examples/src/main/scala/com/criteo/cuttle/examples/HelloWorld.scala

```diff
@@ -45,35 +45,49 @@ object HelloWorld {
           val partitionToCompute = (e.context.start) + "-" + (e.context.end)
 
           // The `sh` interpolation is provided by the local platform. It allows us to
-          // declare a bash script to execute.
-          sh"""
-          echo "Hello for ${partitionToCompute}"
-          echo "Check my project page at https://github.com/criteo/cuttle"
-          sleep 1
-        """.exec()
+          // declare a bash command to execute. If you want use many commands you can combine them
+          // in a for comprehension.
+          for {
+            _ <- sh"echo Hello for $partitionToCompute".exec()
+            _ <- sh"echo Check my project page at https://github.com/criteo/cuttle".exec()
+            completed <- sh"sleep 1".exec()
+          } yield completed
       }
 
     // Our second job is also on hourly job. Nothing special here.
+    // We still can do pretty sophisticated things with sh commands.
+    // For example you can create an executable file and run it.
     val hello2 =
       Job("hello2", hourly(start), "Hello 2") { implicit e =>
-        sh"""
-          echo "Looping for 20 seconds..."
-          for i in {1..20}; do
-            date
-            sleep 1
-          done
-          echo "Ok"
-        """.exec()
+        val scriptName = s"/tmp/run${e.id}"
+        for {
+          _ <- sh"echo Looping for 20 seconds...".exec()
+          _ <- sh"""
```

Comment:
Why doing that instead of an `sh -ce` ? Something like:

```
exec"""sh -ce '
  echo "Looping for 20 seconds..."
  for i in {1..20}; do
    date
    sleep 1
  done
  echo "Ok"
'"""()
```